db/bigtable_eth1.go

```diff
@@ -3426,7 +3433,7 @@ func (bigtable *Bigtable) SaveMethodSignatureImportStatus(status types.MethodSig
 }
 
 // Save a list of method signatures
```

Comment:
**Minor:** Comment now is wrong as this supports both methods and events.

Reply:
fixed